packages/gestalt/src/Popover/Contents.js

```diff
@@ -0,0 +1,133 @@
+// @flow strict
```

Comment:
Copied legacy Contents and converted to function component removing unecessary parts.